### ISSUE TITLE
[Bug?] Problems with UVR-DeEcho-DeReverb

### ISSUE THREAD

**Issue by ShiromiyaG** [NextAudioGen/ultimatevocalremover_api#4]
I tested several models, both MDX, MDXC and VR, but the only model that I had results that were actually worse than the UVR GUI was the UVR-DeEcho-DeReverb. Here is a link to a song comparing UVR-DeEcho-DeReverb and MDX Reverb HQ from the UVR api, along with UVR-DeEcho-DeReverb from the UVR GUI

https://drive.google.com/drive/folders/1_E0PzZRtK4f8Ah3rknJBDitZZvDNnFeO?usp=drive_link

And this is the code I used:

```
MDX = models.MDX(name="Reverb_HQ",  other_metadata={'segment_size': 256,'overlap': 0.75,'mdx_batch_size': 8,'semitone_shift': 0,'adjust': 1.08, 'denoise': False,'is_invert_spec': False,'is_match_frequency_pitch': True,'overlap_mdx': None},device=device, logger=None)
res = MDX(name)
seperted_audio = res["no reverb"]
```
```
Vr = models.VrNetwork(name="UVR-DeEcho-DeReverb", other_metadata={'normaliz': False,'aggressiveness': 0.05,'wav_type_set': "PCM_F32",'window_size': 320,'batch_size': 4,'is_tta': True},device=device, logger=None)
res = Vr(name)
seperted_audio = res["no reverb"]
```
Also, a photo of Audacity comparing Reverb HQ and UVR-DeEcho-DeReverb:
![image](https://github.com/NextAudioGen/ultimatevocalremover_api/assets/86894155/8d34e19b-80e8-4a95-9da7-75d2239142ed)

**Comment by MohannadEhabBarakat** [comment]
Thanks 🤩. I'll check your results and try to figure out why this happened. I'm just curious, did you use any custom parameters in UVR?

**Comment by ShiromiyaG** [comment]
The only thing I changed was the window size, which I set to "320" and the aggression setting, which I set to "5", otherwise everything is in the default setting.

**Comment by MohannadEhabBarakat** [comment]
Thats interesting. The code looks good. And I believe that UVR doesn't use secondary models by default, which means that the results should be identical. I'll check this later around 12th when I come back

**Comment by ShiromiyaG** [comment]
Yes, UVR does not use secondary models by default, and in this case, I did not activate it
